lib/export/cancer_death_common.rb

```diff
@@ -55,6 +59,9 @@ class CancerDeathCommon < DeathFile # rubocop:disable Metrics/ClassLength
     #                       [/WOLFRAM/],
     #                       [/DIDMOAD/],
     #                       [/MULTI/, /ATROPHY/]].freeze
+    # Custom criteria used 2024-10, cf. JIRA NDRS3-337
+    # RD_STRING_PATTERNS = [[/EPIDERMOLYSIS BULLOSA/],
+    #                       [/EPIDERMOLYSIS/]].freeze
```

Comment:
I might be totally misunderstanding the request, but would this match the below string from ticket, if it included the `r` in `epidemolysis`? not sure if it's a ticket typo or not.

"Please can the free text 'epidemolysis bullosa acquisita' be excluded."

Reply:
Yes, it matches the string 'epidemolysis bullosa acquisita'. I ended up removing those manually: if I'd just excluded them in `RD_STRING_PATTERNS`, many of them would still have matched by ICD code and been included in the output.

I decided it was quicker to remove them manually for this one-off extract, than add an extra codepath for exclusions of free text when the ICD code matches.